main.go

```diff
@@ -58,6 +59,7 @@ func init() {
 	flag.BoolVar(&skipForeignKeys, "skip-foreign-keys", false, "skip-foreign-keys: if true, skip creating foreign keys after data migration is complete (ddl statements for foreign keys can still be found in the downloaded schema.ddl.txt file and the same can be applied separately)")
 	flag.StringVar(&sessionJSON, "session", "", "session: specifies the file we restore session state from (used in schema-only to provide schema and data mapping)")
 	flag.BoolVar(&webapi, "web", false, "web: run the web interface (experimental)")
+	flag.StringVar(&dumpFile, "dump-file", "", "location of dump file to process")
```

Comment:
nit; suggest rewording the help text as "dump-file: location of dump file to process" to keep it consistent with other flags defined above

Reply:
done